tests/functional/adapter/test_concurrency.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from pathlib import Path
+from dbt.tests.util import (
+     run_dbt,
+     check_relations_equal,
+     rm_file,
+     write_file
+     )
+from dbt.tests.adapter.concurrency.test_concurrency import (
+     BaseConcurrency,
+     seeds__update_csv
+     )
+
+class TestConcurrencyImpala(BaseConcurrency):
+     def test_concurrency_impala(self, project):
+         run_dbt(["seed", "--select", "seed"])
+         results = run_dbt(["run"], expect_pass=False)
+         assert len(results) == 7
```

Comment:
Why 7.  Can you point me to the base test case

Reply:
https://github.com/dbt-labs/dbt-core/blob/b0f8d3d2f141a46b59e5cce2d8a4f95cf9b6f530/tests/adapter/dbt/tests/adapter/concurrency/test_concurrency.py#L296-L302